dashboard/views.py

```diff
@@ -0,0 +1,18 @@
+from django.shortcuts import render
+from OneApply.constants import UserType
+
+
+def student(request):
+    context = {
+        "constant_ut_student": UserType.STUDENT,
+        "constant_ut_adminStaff": UserType.ADMIN_STAFF,
+    }
+    return render(request, "dashboard/student.html", context)
+
+
+def admissionstaff(request):
+    context = {
+        "constant_ut_student": UserType.STUDENT,
```

Comment:
Is there a particular reason why the student user type is being mentioned in the admissions staff view?